crates/router/src/compatibility/stripe/setup_intents/types.rs

```diff
@@ -108,11 +115,65 @@ impl From<Shipping> for payments::Address {
         }
     }
 }
+
+#[derive(PartialEq, Eq, Deserialize, Clone, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum StripePaymentMethodOptions {
+    Card {
+        request_three_d_secure: Option<Request3DS>,
+        mandate_options: Option<MandateOption>,
+    },
+}
+
+#[derive(Default, Eq, PartialEq, Serialize, Deserialize, Clone, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum Request3DS {
+    #[default]
+    Automatic,
+    Any,
+}
+
+impl ForeignFrom<Option<Request3DS>> for api_models::enums::AuthenticationType {
```

Comment:
I guess this implementation is there in `payment_intents/types.rs`. Check if that solves the use case here.